.ci/run_e2e_tests.sh

```diff
@@ -1,5 +1,14 @@
 SCRIPTS_PATH=$(cd $(dirname "${BASH_SOURCE[0]}"); pwd)
 SRC_PATH=$(dirname "${SCRIPTS_PATH}")
+
+# Early auth to avoid limit rating
+# If Docker Credentials are not available skip the Byzantine Tests
```

Comment:
This seems to be similar to
https://github.com/ava-labs/avalanchego/pull/725/files#diff-5094e7b250ccea461e290b48addfa38d35af80538dbd9c313a91dafb2e5e0c4cR11?#728 

cc @aaronbuchwald